rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -93,21 +93,24 @@ def _toJSON(mol):
 
 
 def _wrapHTMLIntoTable(html):
-  return InteractiveRenderer.injectHTMLHeaderBeforeTable(
+  return InteractiveRenderer.injectHTMLFooterAfterTable(
     f'<div><table><tbody><tr><td style="width: {molSize[0]}px; ' +
     f'height: {molSize[1]}px; text-align: center;">' + html.replace(" scoped", "") +
     '</td></tr></tbody></table></div>')
 
 
 def _toHTML(mol):
+  def escape_special_chars(s):
```

Comment:
Why not use python's builtin `html.escape()` for this?

Reply:
Done. I actually also need to escape the ampersand after everything else has been escaped, so I still need `escape_special_chars()`, but I can let `html.escape()` take care of most of the work.